### PR TITLE
Fix jar name and download

### PR DESCRIPTION
**Pull request by Martmists-GH**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -610,10 +610,10 @@ Please see the [Dokka Maven example project](https://github.com/JetBrains/kotlin
 
 ### Using the Command Line
 
-To run Dokka from the command line, download the [Dokka CLI runner](https://github.com/Kotlin/dokka/releases/download/1.4.0-rc/dokka-cli.jar).
+To run Dokka from the command line, download the [Dokka CLI runner](https://github.com/Kotlin/dokka/releases/download/1.4.0-rc/dokka-cli-1.4.0-rc.jar).
```
Comment (sellmair) [inline README.md:613]: I think there is a `v` missing before the version ☺️ 
`https://github.com/Kotlin/dokka/releases/download/v1.4.0-rc/dokka-cli-1.4.0-rc.jar`

Comment (floscher) [inline README.md:613]:
```suggestion
To run Dokka from the command line, download the [Dokka CLI runner](https://github.com/Kotlin/dokka/releases/download/v1.4.0-rc/dokka-cli-1.4.0-rc.jar).
```

**Comment by sellmair** [comment]
Could you please rebase on lastest master and squash the commits into one? ☺️

**Comment by IgnatBeresnev** [comment]
The root README.md has been re-written from scratch, so I'm closing the PR as outdated. Thank you for your effort though!

**Pull request closed without merging**